seth.go

```diff
@@ -237,6 +238,14 @@ func (i *Int) Msgsize() int {
 // Uint64 is a uint64 that marshals as a hex-encoded number.
 type Uint64 uint64
 
+func ParseUint(s string) (*Int, error) {
```

Comment:
You can replace this function with this:
```go
func ParseInt(s string) (*Int, error) {
  n := new(Int)
  if err := n.FromString(s); err != nil {
    return nil, err
  }
  return n, nil
}
```
The name `ParseInt` is preferable, since the type it returns is `*Int`. Also, please move this to somewhere under the type declaration for `Int`. Probably directly under `NewInt` would be most appropriate.